sdk/src/platform/Platform.ts

```diff
@@ -549,7 +549,12 @@ export default class Platform extends EventEmitter {
         if (this._discoveryInitPromise) {
             await this._discoveryInitPromise;
         }
-        const discoveryData = await this._discovery.initialData();
+        let discoveryData = await this._discovery.initialData();
+        if (!discoveryData) {
+            this._discoveryInitPromise = this.initDiscovery(); // request new init data after refresh error and cache cleaned
+            await this._discoveryInitPromise;
+            discoveryData = await this._discovery.initialData();
+        }
```

Comment:
`await this._discovery.initialData()` should do this functionality behind the scenes. `await this._discoveryInitPromise` is referred from too many places.

Reply:
Thanks for reviewing. It is updated. 
We want to fetch new initial discovery data again at login if initial discovery is not fetched successfully when app start.

Reply:
@kirill-konshin can you help to review again